controller/src/beerocks/master/db/db.h

```diff
@@ -174,6 +174,9 @@ class db {
 
     int get_node_channel(std::string mac);
 
+    bool set_hostap_operating_class(std::string mac, uint8_t operating_class);
+    int get_hostap_operating_class(std::string mac);
```

Comment:
If you use a string argument, it probably should be a const reference.
@arnout maybe we should use sMacAdde? What do you think?
Second, I would really like to get rid of "hostap" terminology and use "radio" instead.

Reply:
> If you use a string argument, it probably should be a const reference.
> @arnout maybe we should use sMacAdde? What do you think?

Yes please, for any new functionality, use only sMacAddress.

Also, any non-scalar parameter (which std::string definitely is) should use const-ref (cfr. [Coding style, functions, 2nd paragraph](https://github.com/prplfoundation/prplMesh/blob/master/CODINGSTYLE.md#functions)).

> Second, I would really like to get rid of "hostap" terminology and use "radio" instead.

I agree on the principle. However, currently "radio" is used (almost) nowhere, so using it for this would just create more confusion. So I created a new task #436.

Reply:
Done.